app/src/main/java/com/amaze/filemanager/AmazeFileManagerModule.java

```diff
@@ -17,5 +18,6 @@
     @Override
     public void registerComponents(Context context, Glide glide, Registry registry) {
         registry.prepend(String.class, Drawable.class, new ApkImageModelLoaderFactory(context.getPackageManager()));
+        registry.prepend(String.class, String.class, new CloudIconModelFactory(context));
```

Comment:
That second `String.class` should probably be `Bitmap.class` (also, check the generic type inference missing in `CloudIconModelLoader`).